sql/mob_pools.sql

```diff
@@ -5860,6 +5860,7 @@ INSERT INTO `mob_pools` VALUES (5840,'Champion_Crawler','Champion_Crawler',79,0x
 INSERT INTO `mob_pools` VALUES (5841,'Hovering_Hotpot','Hovering_Hotpot',175,0x00009C0100000000000000000000000000000000,5,1,7,240,100,0,1,0,0,2,0,0,0,0,0,0,36,1,0,998);
 INSERT INTO `mob_pools` VALUES (5842,'Nargun','Nargun',135,0x0000B00100000000000000000000000000000000,1,1,11,320,100,0,1,0,0,2,0,32,0,0,0,0,0,1,0,1005);
 INSERT INTO `mob_pools` VALUES (5843,'Ignamoth','Ignamoth',253,0x0000100700000000000000000000000000000000,1,1,7,240,100,0,1,1,1,2,39,0,311,131,0,0,0,1,0,1006);
+INSERT INTO `mob_pools` VALUES (5844,'Ankabut','Ankabut',235,0x0000360100000000000000000000000000000000,1,1,7,240,100,1,0,0,1,0,0,0,1,131,0,0,0,1,0,235);
```

Comment:
mobType needs to be 2 for Notorious Monster.
Does not link, according to both bg-wiki and ffxiclopedia.
Both wikis also mention that it only uses Acid Spray so this needs to use a different skill list that only includes Acid Spray.